src/Configurator.php

```diff
@@ -15,18 +15,6 @@
     const SETTING_INFLECTORS_KEY             = 'inflectors_key';
     const SETTING_DEFAULT_SINGLETON_SERVICES = 'default_singleton_services';
 
-    const FILE_READERS = [
-        '.json' => FileReader\JSONFileReader::class,
-        '.php'  => FileReader\PHPFileReader::class,
-        '.yaml' => FileReader\YAMLFileReader::class,
-        '.yml'  => FileReader\YAMLFileReader::class,
-    ];
-
-    const CONTAINER_ADAPTERS = [
-        \League\Container\Container::class => League\LeagueContainerAdapter::class,
-        \Pimple\Container::class           => Pimple\PimpleContainerAdapter::class,
-    ];
```

Comment:
Indeed it is, but I think HHVM support would be an acceptable reason to for a new major release right? I think it's also very unlikely that anyone is currently accessing these constants directly so it shouldn't affect anyone.

There are also some BC breaking changes I was hoping to make to the API in a major release at some point, but maybe that can be in a release - I can elaborate further if you are interested though?